Sources/Fuzzilli/FuzzIL/Program.swift

```diff
@@ -89,6 +89,18 @@ public final class Program {
     public func clearParent() {
         parent = nil
     }
+
+    /// Simple comparison by size, used to sort compiled corpora prior to importing.
+    public static func < (lhs: Program, rhs: Program) -> Bool {
+        return lhs.size < rhs.size
+    }
+
+    /// We consider two programs to be equal if their code is equal
```

Comment:
I'd prefer keeping this code only in FuzzilliTests since it shouldn't be relied on by any other code. Can you use the [`sort(by:)`](https://developer.apple.com/documentation/swift/array/2296801-sort) method to sort the input programs without having Program be Comparable?